app/angular/src/server/options.ts

```diff
@@ -1,4 +1,5 @@
-import packageJson from '../../package.json';
+// tslint:disable-next-line: no-var-requires
+const packageJson = require('../../package.json');
```

Comment:
why was this changed?

Reply:
Because this is outside the typescript root folder,

import doesn't work then, require (for some reason) does.

Reply:
til, still seems like an unsatisfying solution, i guess adding the `app/angular` to the `rootDirs` option wont work?

Reply:
no, that causes far more grief.